source/_components/notify.hipchat.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+title: "HipChat"
+description: "Instructions how to add HipChat notifications to Home Assistant."
+date: 2017-08-10 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hipchat.png
+ha_category: Notifications
+ha_release: "0.51"
+---
+
+
+The `hipchat` platform allows you to deliver notifications from Home Assistant to [HipChat](https://hipchat.com/).
+
+You need to obtain a [HipChat API token](https://developer.atlassian.com/hipchat/guide/hipchat-rest-api/api-access-tokens#APIaccesstokens-Usergeneratedtokens) to be able to send notifications.
+
+To enable the HipChat notification in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+notify:
+  - name: NOTIFIER_NAME
+    platform: hipchat
+    token: ABCDEFGHJKLMNOPQRSTUVXYZ
+    room: '1234567'
+```
+
+Configuration variables:
+
+- **name** (*Optional*): Setting the optional parameter `name` allows multiple notifiers to be created. The default value is `notify`. The notifier will bind to the service `notify.NOTIFIER_NAME`.
+- **token** (*Required*): The HipChat API token to use for sending HipChat notifications.
+- **room** (*Required*): The default room to post to if no room is explicitly specified when sending the notification message.
+- **color** (*Optional*): Setting color will override the default color for the notification. By default not setting this will post to HipChat using the default color yellow. Valid options are 'yellow', 'green', 'red', 'purple', 'gray', 'random'.
+- **notify** (*Optional*): Setting notify will override the default notify (blink application icon, chime, or otherwise call attention) setting for the notification. By default this is false. Valid options are true and false.
```

Comment:
It should be true / false (bool) not 'true' / 'false' (str), so I didn't put in quotes. I can change it, or is there a better third option for making it clear without implying str?